tests/feature/test_outline.py

```diff
@@ -166,3 +166,71 @@ def test_outlined_feature(request):
         ["fruits"],
         [[u"oranges"], [u"apples"]],
     )
+
+
+def test_outline_with_escaped_pipes(testdir):
```

Comment:
The "when" and "then" steps are doing nothing, but we could improve the test by having "string1" being the string one under test, and string2 as the base64 representation of string1, and the "then" step can be something like "then <string2> should be the base64 encoding of <string1>.

The feature file would look like this (note the use of `r""" """` string, it allows to use less backslashes):
```
r"""\
    Feature: Outline With Special characters
        Scenario Outline: Outline with escaped pipe character
            Given we have strings <string1> and <string2>
            Then <string2> should be the base64 encoding of <string1>
            Examples:
            | string1        | string2          |
            | bork           | Ym9yaw==         |
            | \|bork         | fGJvcms=         |
            | bork \|        | Ym9yayB8         |
            | bork\|\|bork   | Ym9ya3x8Ym9yaw== |
            | \|             | fA==             |
"""
```

Let's also have these examples, where we check some edge cases with the escaping:
```
| bork \\\| | Ym9yayBcfA== |
| bork \\   | Ym9yayBc     |
```

Reply:
Great idea! Just implemented and pushed it. The edge-case with 3 backslashes had a different base64 encoding actually, because the 3rd backslash is consumed by `split_line()`.